ACPs/77-reinventing-subnets/README.md

```diff
@@ -127,18 +149,27 @@ type EvictSubnetValidatorTx struct {
 The `Message` field in the above transaction must be an Avalanche Warp Message using the `AddressedCall` format with the below payload. A `typeID` for distinguishing between the different payload types on the P-Chain must be included prior to this ACP being considered "Implementable".
 
 ```text
-+-----------+----------+----------+
-|      txID : [32]byte | 32 bytes |
-+-----------+----------+----------+
-                       | 32 bytes |
-                       +----------+
++----------+----------+----------+
+| subnetID : [32]byte | 32 bytes |
++----------+----------+----------+
+|   nodeID : [32]byte | 32 bytes |
++----------+----------+----------+
+|   weight :   uint64 |  8 bytes |
++----------+----------+----------+
+|    nonce :   uint64 |  8 bytes |
++----------+----------+----------+
+                      | 80 bytes |
+                      +----------+
 ```
 
+- `subnetID`, `nodeID`, and `weight` are for the Subnet Validator being modified
+- `nonce` is a strictly increasing number that denotes the latest validator weight update and provides replay protection for this transaction. The P-Chain will store the largest `nonce` that has been used to modify the `nodeID`'s weight. The `nonce` is not required to be incremented by `1` with each successive validator weight update. If the `nonce` is equal to MaxUint64, `weight` is required to be `0` to prevent Subnets from being unable to remove `nodeID` in the future.
```

Comment:
If this condition is not maintained, what happens? Is the tx included and pays fees or is it not able to be included?

Reply:
It isn't included